microprofile/WFLY-19588_microprofile_platform_7.adoc

```diff
@@ -0,0 +1,201 @@
+
+=== Nice-to-Have Requirements
+// Requirements in this section do not have to be met to merge the proposed functionality.
+// Note: Nice-to-have requirements that don't end up being implemented as part of
+// the work covered by this proposal should be moved to the 'Future Work' section.
+None
+
+=== Non-Requirements
+// Use this section to explicitly discuss things that readers might think are required
+// but which are not required.
+None
+
+=== Future Work
+// Use this section to discuss requirements that are not addressed by this proposal
+// but which may be addressed in later proposals.
+
+== Backwards Compatibility
+// Does this enhancement affect backwards compatibility with previously released
+// versions of WildFly?
+// Can the identified incompatibility be avoided?
+Where one of the specifications being upgraded is not backwards compatible, it should be pointed out in the analysis of the individual specification upgrade.
+
+=== Default Configuration
+Where one of the specifications being upgraded changes the default configuration, it should be pointed out in the analysis of the individual specification upgrade.
+
+=== Importing Existing Configuration
+
+Where one of the specifications being upgraded changes how it works with existing configuration, it should be pointed out in the analysis of the individual specification upgrade.
+
+=== Deployments
+
+Deployments should be able to use the full MicroProfile Platform 7 feature set.
+
+
+== Admin Clients
+
+Where one of the specifications being upgraded changes its configuration so that extra work is needed in the CLI or HAL, it should be pointed out in the analysis of the individual specification upgrade.
+
+
+== Security Considerations
+
+////
+Identification if any security implications that may need to be considered with this feature
+or a confirmation that there are no security implications to consider.
+////
+I don't believe there are any security implications to consider.
+
+== Test Plan
+* TCKs will be updated to the latest corresponding version. Having checked the upgrades to the individual specifications, the respective TCKs tests cover the added/changed functionality, apart from the exceptions listed below.
+* The TCKs will run at default stabiilty level
+* Where one of the specifications being upgraded needs additional testing beyond what is done by the TCK, it should be pointed out in the analysis of the individual specification upgrade.
+
+== Community Documentation
+* Community documentation for the subsystems in question will be updated to reflect the new versions, and any potential differences in behaviour.
+
+== Release Note Content
+MicroProfile specifications have been updated to the versions that are part of MicroProfile Platform 7.0. (See the table in the link:#overview[Overview] if exact versions are needed)
+
+MicroProfile OpenTracing has been removed from the MicroProfile 6 platform, in favour of MicroProfile Telemetry.
```

Comment:
Don't we know this already? I mean, we removed the integration when we upgraded the specs to 6.1, right? Or am I missing something?

Reply:
Thanks for fixing. Feel free to resolve.